main/test/smoke.test.js

```diff
@@ -0,0 +1,7 @@
+const assert = require('node:assert')
+
+describe('smoke test', function () {
```

Comment:
@bajtos could you take a look at the github actions errors below?

Reply:
Wow, that error message was super helpful.

```
Error: main/test/smoke.test.js(3,1): error TS2582: Cannot find name 'describe'. 
  Do you need to install type definitions for a test runner? 
  Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.
```

Fixed in 8f3dc2b